src/Controls/src/Core/Compatibility/Handlers/Shell/iOS/ShellPageRendererTracker.cs

```diff
@@ -796,7 +796,14 @@ void SearchButtonClicked(object sender, EventArgs e)
 
 		void SetSearchBarIcon(UISearchBar searchBar, ImageSource source, UISearchBarIcon icon)
 		{
-			source.LoadImage(source.FindMauiContext(), image =>
+			var mauiContext = MauiContext ?? Application.Current?.FindMauiContext();
```

Comment:
this could be directly `icon.FindMauiContext(true)`, where the bool parameter indicates to fallback to the App MauiContext.

Reply:
@jsuarezruiz, I have modified it as suggested.